src/GiftedChat.tsx

```diff
@@ -588,6 +589,19 @@ class GiftedChat<TMessage extends IMessage = IMessage> extends React.Component<
   }
 
   onKeyboardWillShow = (e: any) => {
+    // Focus the text input only if it was focused before showing keyboard
```

Comment:
Can you please extract this code into a function of that class with an explicit name (and remove comments)?
Thanks